sdk/storage/storage-blob/samples/typescript/iterators.ts

```diff
@@ -56,8 +49,8 @@ async function main() {
     // Create a blob
     let content = "hello";
     let blobName = "newblob" + new Date().getTime();
-    let blobClient = BlobClient.fromContainerClient(containerClient, blobName);
-    let blockBlobClient = BlockBlobClient.fromBlobClient(blobClient);
+    let blobClient = containerClient.createBlobClient(blobName);
+    const blockBlobClient = blobClient.createBlockBlobClient();
```

Comment:
Maybe fix others as well if they can be `const` to be consistent.

Reply:
Oh yes. Done